mgmtsystem/ec2.py

```diff
@@ -16,6 +21,12 @@
 )
 
 
+def regions(regionlist, regionname):
```

Comment:
Regions have different endpoints in every service. So I have to get regions for every service.